src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -341,3 +341,71 @@ def subdata(meta, i, n, m, subdata, submask, expected, loc):
     plt.savefig(meta.outputdir+fname, dpi=300)
     if not meta.hide_plots:
         plt.pause(0.1)
+
+
+def driftypos(data, meta):
+    '''Plot the spatial jitter. (Fig 3601)
+
+    Parameters
+    ----------
+    data : Xarray Dataset
+        The Dataset object.
+    meta : eureka.lib.readECF.MetaClass
+        The metadata object.
+
+    Returns
+    -------
+    None
+    
+    Notes
+    -----
+    History:
+    
+    - 2022-07-11 Caroline Piaulet
+        First version of this function
+    '''
+    plt.figure(3601, figsize=(8, 4))
+    plt.clf()
+    plt.plot(np.arange(meta.n_int),
+             np.array(data["driftypos"]), '.')
```

Comment:
You shouldn't need to do `np.array` - if anything you can do `data["driftypos"].values` if just `data["driftypos"]` doesn't work

Reply:
done